ws-nextjs-app/pages/[service]/articles/handleArticleRoute.ts

```diff
@@ -100,6 +95,14 @@ export default async (context: GetServerSidePropsContext) => {
 
   const { article, secondaryData } = data?.pageData || {};
 
+  const isArticleOlderThanSixHours = Date.now() - article.metadata.lastPublished > 21600000;
```

Comment:
```suggestion
  const isArticleOlderThanSixHours = Date.now() - article.metadata?.lastPublished > 21600000;
```

idk if `lastPublished` can be missing, is it always present when an article has never been modified?